From f317cb00bcd2e6e8ae1406b2bcdab742114c73f9 Mon Sep 17 00:00:00 2001
From: Alessandro Astone <ales.astone@gmail.com>
Date: Mon, 29 Aug 2022 01:50:24 +0200
Subject: [PATCH 1/2] Render shadows in the client by default

We rely on applications drawing their own shadows so that in multi-windows
mode there is some space to grab their edges for resizing

Change-Id: I775924c2ea84e28c20c29273e613dc1e7391c76d
---
 core/java/com/android/internal/policy/PhoneWindow.java          | 2 +-
 .../core/java/com/android/server/wm/WindowManagerService.java   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/java/com/android/internal/policy/PhoneWindow.java b/core/java/com/android/internal/policy/PhoneWindow.java
index fb3bb0d8346c..0e0d962990c6 100644
--- a/core/java/com/android/internal/policy/PhoneWindow.java
+++ b/core/java/com/android/internal/policy/PhoneWindow.java
@@ -347,7 +347,7 @@ public class PhoneWindow extends Window implements MenuBuilder.Callback {
         super(context);
         mLayoutInflater = LayoutInflater.from(context);
         mRenderShadowsInCompositor = Settings.Global.getInt(context.getContentResolver(),
-                DEVELOPMENT_RENDER_SHADOWS_IN_COMPOSITOR, 1) != 0;
+                DEVELOPMENT_RENDER_SHADOWS_IN_COMPOSITOR, 0) != 0;
     }
 
     /**
diff --git a/services/core/java/com/android/server/wm/WindowManagerService.java b/services/core/java/com/android/server/wm/WindowManagerService.java
index 688ee8e03919..c5475f102c88 100644
--- a/services/core/java/com/android/server/wm/WindowManagerService.java
+++ b/services/core/java/com/android/server/wm/WindowManagerService.java
@@ -934,7 +934,7 @@ public class WindowManagerService extends IWindowManager.Stub
 
     private void setShadowRenderer() {
         mRenderShadowsInCompositor = Settings.Global.getInt(mContext.getContentResolver(),
-                DEVELOPMENT_RENDER_SHADOWS_IN_COMPOSITOR, 1) != 0;
+                DEVELOPMENT_RENDER_SHADOWS_IN_COMPOSITOR, 0) != 0;
     }
 
     PowerManager mPowerManager;
-- 
2.37.2

